src/python/test/test_dxclient.py

```diff
@@ -9029,8 +9029,10 @@ def test_basic(self):
                          "distribution": "Ubuntu",
                          "release": "14.04" }
         })
-        o2 = run("dx run {} --batch-tsv={}".format(applet["id"], arg_table))
-        print(o2)
+        job_id = run("dx run {} --batch-tsv={} --yes --brief"
+                     .format(applet["id"], arg_table)).strip()
+        job_desc = dxpy.api.job_describe(job_id)
+        self.assertEquals(job_desc["executableName"], 'copy_all')
```

Comment:
Could one more assertion be added here, that the input values were passed as expected, something like [here](https://github.com/dnanexus/dx-toolkit/blob/2a2d72273aa71f9a918fcbcdeb7ce70bdcdc6b65/src/python/test/test_dxclient.py#L2098) but probably simpler.